lib/Transforms/IPO/StructFieldCacheAnalysis.cpp

```diff
@@ -385,18 +469,30 @@ static void performIRAnalysis(Module &M,
   DEBUG_WITH_TYPE(DEBUG_TYPE_IR, StructManager->debugPrintAnnotatedModule());
 }
 
+static void applyFilters(StructFieldAccessManager *StructManager) {
+  StructManager->summarizeFunctionCalls();
+  StructManager->applyFiltersToStructs();
+  StructManager->printStats();
```

Comment:
Do you really always want to call printStats, or should this be guarded by some kind of debugging or output flag?